src/modules/mc_att_control/mc_att_control.hpp

```diff
@@ -277,6 +279,9 @@ class MulticopterAttitudeControl : public ModuleBase<MulticopterAttitudeControl>
 		(ParamInt<px4::params::MC_AIRMODE>) _param_mc_airmode
 	)
 
+	param_t _param_handle_vtol_type{0};
```

Comment:
```suggestion
	param_t _param_handle_vtol_type{PARAM_INVALID};
```

Reply:
@dagar Thanks, I was missing that. I just saw that it was a unit and made the false assumption that 0 was invalid.